packages/react-examples/.storybook/preview.js

```diff
@@ -28,7 +28,11 @@ if (
   addDecorator(withCompatThemeProvider);
   addDecorator(withStrictMode);
 }
-if (['react-avatar', 'react-badge', 'react-link', 'react-image', 'react-menu'].includes('PACKAGE_NAME')) {
+if (
```

Comment:
Doesn't need to happen now, but it would be nice to add a step in the create-package script to automatically add converged packages to this list (or switch it to use some other heuristic)